src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -157,4 +240,42 @@ components:
         "marginalCarbonIntensityValue": 750.0,
         "embodiedEmissionsValue": null,
         "functionalUnitValue": null 
-      }
\ No newline at end of file
+      }
+    
```

Comment:
Nit - 'notBefore' and 'notAfter' sounds a bit confusing and negative too. Would it be better to have something like 'fromDate' or fromTime or something else? Not a big deal, just brainstorming.

Reply:
Yea, I went back and forth on this.  I agree that the naming could be more intuitive. I'll propose something and ping you for more feedback :)

Reply:
@pritipath updated! Let me know what you think

Reply:
looks good!